src/reputation/migrations/0065_auto_20221103_1610.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2 on 2022-11-03 16:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
I did not write this one. It showed up when I `makemigrations`

Reply:
Let's delete this file.